src/frontend/api.py

```
@@ -0,0 +1,60 @@
+import requests
+
+BACKEND_URL = "http://localhost:8000"
+
+def check_health():
+
+    return requests.get(
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)